src/scss/components/_home.scss

```diff
@@ -0,0 +1,50 @@
+@import '../variables';
+
+.home{
+  height: 75vh;
+  width: 55vw;
+  margin: 50px auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  background-color: $middle;
+  border-radius: 100px;
+
+
+
+  &__link{
+    //margin: 25px 0;
+    //text-underline-offset: -1089px;
+    font-size: 42px;
+    font-weight: 600;
+    letter-spacing: 0.2em;
```

Comment:
this tells me that this code was copy pasted , please try to write code on your own

Reply:
I was looking for great solutions for buttons and found this button, which I later modify to suit my code

Reply:
of course copied, from a neighbouring component, because it is similar to the main component